kubelogic.gemspec

```diff
@@ -23,4 +23,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.16"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "rails", "~> 3.0"
```

Comment:
Why development dependency?
And why rails 3.0?

Reply:
What rails version should be? I think greater that 3.0